src/main/java/org/sonarlint/intellij/core/SonarLintEngineManager.java

```diff
@@ -141,5 +143,6 @@ public void dispose() {
       standalone.stop();
       standalone = null;
     }
+    Loggers.setTarget(null);
```

Comment:
I don't know why, but I don't like this change. We can see in the next sprint if we can do better.

Reply:
:+1: sure we can do better